scripts/install.sh

```
@@ -20,7 +20,7 @@ DATE=$(date +"%Y%m%d-%H%M%S")
 
 OS_TYPE=$(grep -w "ID" /etc/os-release | cut -d "=" -f 2 | tr -d '"')
 ENV_FILE="/data/coolify/source/.env"
-VERSION="20"
+VERSION="21"
```

Shellcheck (0.10.0)

[warning] VERSION appears unused. Verify use (or export if used externally).

(SC2034)